crates/corro-agent/src/api/public/mod.rs

```diff
@@ -474,6 +481,13 @@ pub async fn api_v1_queries(
     let (data_tx, mut data_rx) = channel(512);
 
     tokio::spawn(async move {
+        if !pms.ndjson {
+            if let Err(e) = tx.send_data(bytes::Bytes::from_static(b"[")).await {
```

Comment:
It would be cleaner to implement a `tokio_util::codec::{Encoder,Decoder}` for this format!

I wouldn't hold back this PR for that though!